schema/artist/index.js

```diff
@@ -462,13 +472,15 @@ const ArtistType = new GraphQLObjectType({
               is_reference: true,
               solo_show: false,
               at_a_fair: false,
+              visible_to_public: false,
               size: options.size,
             }),
             // Fair booths
             gravity('related/shows', {
               artist_id: id,
               sort: '-end_at',
               at_a_fair: true,
+              visible_to_public: false,
```

Comment:
Yes please. _Now_ I understand why we that API endpoint gets so much traffic (up to ~5K rpm!).

<img width="533" alt="screen shot 2016-12-08 at 8 41 59 pm" src="https://cloud.githubusercontent.com/assets/28120/21034608/175e41f8-bd87-11e6-8866-692a8d4f315d.png">

We have the equivalent of 3 servers serving just these requests some days.